core/build.sbt

```diff
@@ -1,5 +1,5 @@
 
 libraryDependencies ++= Seq(
-  "org.scalaz" %% "scalaz-core"       % "7.1.2",
-  "org.scalaz" %% "scalaz-concurrent" % "7.1.2"
+  "org.scalaz" %% "scalaz-core"       % "7.2.8",
```

Comment:
This wont fly im afraid. We need scalaz 7.1, so we'll have to do a matrix build like we're doing on the ermine parser: https://github.com/Verizon/ermine-parser/blob/master/.travis.yml#L3 - this will then produce the various combinations people would want.